index.js

```diff
@@ -309,6 +309,43 @@ class Client {
         fileKey    = filePath.replace(_this.clientPath, '').substr(1).replace(/\\/g, '/'),
         urlRoot    = regionToUrlRootMap(_this.region);
 
+    let distRoot = path.join(
+      _this.serverless.config.servicePath,
+      _this.serverless.service.custom.client.distributionFolder || path.join('client', 'dist'),
+      'DUMMY'
+    ).replace(/\\/g, '/');
+    distRoot = distRoot.substr(0, distRoot.length - 5);
```

Comment:
Is this to remove 'DUMMY' from the end?

Can you explain why this is needed? There should probably be a more elegant solution here?

Reply:
Yeah this was hacky...I'll come up with something better